From 67f233597b27c31c74ec624e22ac1fe0017c0a45 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 04 Mar 2021 21:10:47 +0200
Subject: [PATCH] [SW-31170] greco: replace goya2 to greco

As goya2 is renamed to greco, this patch add changes to replace
conditions in makefile from goya2 to greco.

Change-Id: I0c50dd7d725eb63fff4a94f6e2c1390c54cafde2
---

diff --git a/package/preboot/preboot.mk b/package/preboot/preboot.mk
index be59fcc..83d9a87 100644
--- a/package/preboot/preboot.mk
+++ b/package/preboot/preboot.mk
@@ -2,7 +2,7 @@
 PREBOOT_SITE = preboot
 PREBOOT_LICENSE = unclear
 
-ifneq ($(HL_DEV),goya2_arc)
+ifneq ($(HL_DEV),greco_arc)
 PREBOOT_DEPENDENCIES = dtc
 endif
 
@@ -26,9 +26,9 @@
 PREBOOT_INSTALL_IMAGES = YES
 
 define PREBOOT_INSTALL_IMAGES_CMDS
-	if [ "$(HL_DEV)" == "goya2_arc_gnu" ]; then \
+	if [ "$(HL_DEV)" == "greco_arc_gnu" ]; then \
 		$(INSTALL) -D -m 0644 $(@D)/bin/preboot.bin $(BINARIES_DIR)/preboot.bin; \
-	elif [  "$(HL_DEV)" == "goya2_arc" ]; then \
+	elif [  "$(HL_DEV)" == "greco_arc" ]; then \
 		$(INSTALL) -D -m 0644 $(@D)/bin/ppboot.bin $(BINARIES_DIR)/ppboot.bin; \
 		$(INSTALL) -D -m 0644 $(@D)/bin/pcie_phy_fw.bin $(BINARIES_DIR)/pcie_phy_fw.bin; \
 	else \
